certbot/tests/cert_manager_test.py

```diff
@@ -8,7 +8,6 @@
 import configobj
 import mock
 
-from certbot import configuration
 from certbot.storage import ALL_FOUR
 
 class CertManagerTest(unittest.TestCase):
```

Comment:
I don't care if you make this change, but you can also write a method like:

```
self._list_certs(*args, **kwargs):
    from certbot.cert_manager import list_certs
    return list_certs(*args, **kwargs)
```

So you don't have to import `cert_manager` at the top of every test. This is arguably even slightly better as you can't modify `cert_manager` other than through `mock` until the very last second before you call it. You don't have to fool around with `*args, **kwargs` if you don't want. If you make a separate class to test each function, you could even call this method `_call`.